tide_core.install

```diff
@@ -894,3 +894,17 @@ function tide_core_update_8035() {
     }
   }
 }
+
+/**
+ * Add field storage for field_display_headings.
+ */
+function tide_core_update_8036() {
+  $config_path = drupal_get_path('module', 'tide_core') . '/config/install';
+  $source = new FileStorage($config_path);
+  $field_storage = 'field.storage.node.field_node_display_headings';
+  // Obtain the storage manager for field storage bases.
+  // Create a new field from the yaml configuration and save.
+  \Drupal::entityManager()->getStorage('field_storage_config')
+    ->create($source->read($field_storage))
```

Comment:
as we install each tide_core_update_{id} during installation of tide_core module. and `field_storage_config` doesn't allow to install it again 2 times. so you should check if `field_node_display_headings` exist or not in running time.
 so , I think you might want to check with :
```
\Drupal::entityManager()->getStorage('field_storage_config')->load('field_node_display_headings');
if null
  install
else
  skip
```